docs/content/v0.20/examples.md

```diff
@@ -0,0 +1,588 @@
+kind: ClusterRoleBinding
+metadata:
+  name: klusterlet-appwrappers-access
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: appwrappers-access
+subjects:
+- kind: ServiceAccount
+  name: klusterlet-work-sa
+  namespace: open-cluster-management-agent
+EOF
+done
+```
+
+This step will be eventually automated, see [this issue](https://github.com/kubestellar/kubestellar/issues/1542)
+for more details.
+
+Apply the appwrapper CRD to wds2:
+
+```shell
+kubectl --context wds2 apply -f https://raw.githubusercontent.com/project-codeflare/multi-cluster-app-dispatcher/v1.33.0/config/crd/bases/mcad.ibm.com_appwrappers.yaml
+```
+
+Now apply an appwrapper CR to wds2:
+
+```shell
+kubectl --context wds2 apply -f  https://raw.githubusercontent.com/project-codeflare/multi-cluster-app-dispatcher/v1.33.0/test/yaml/0001-aw-generic-deployment-3.yaml
+```
+
+Label the CRD and the CR:
+
+```shell
+kubectl --context wds2 label crd appwrappers.mcad.ibm.com app.kubernetes.io/part-of=my-appwrapper-app
+kubectl --context wds2 label appwrappers 0001-aw-generic-deployment-3 app.kubernetes.io/part-of=my-appwrapper-app
+```
+
+Finally, apply the placement:
+
+```shell
+kubectl --context wds2 apply -f - <<EOF
+apiVersion: edge.kubestellar.io/v1alpha1
+kind: Placement
+metadata:
+  name: aw-placement
+spec:
+  clusterSelectors:
+  - matchLabels: {"location-group":"edge"}
+  downsync:
+  - objectSelectors:
+    - matchLabels: {"app.kubernetes.io/part-of":"my-appwrapper-app"}
+EOF
+```
+
+Check that the app wrapper has been delivered to both clusters:
+
+```shell
+kubectl --context cluster1 get appwrappers
+kubectl --context cluster2 get appwrappers
+```
+
+## Scenario 3 - multi-cluster workload deployment with helm
+
+This scenario proceeds from the state established by the [common setup](#common-setup).
+
+Create a placement for the helm chart app:
+
+```shell
+kubectl --context wds1 apply -f - <<EOF
+apiVersion: edge.kubestellar.io/v1alpha1
+kind: Placement
+metadata:
+  name: postgres-placement
+spec:
+  clusterSelectors:
+  - matchLabels: {"location-group":"edge"}
+  downsync:
+  - objectSelectors:
+    - matchLabels: {
+      "app.kubernetes.io/managed-by": Helm,
+      "app.kubernetes.io/instance": postgres}
+EOF
+```
+
+Note that helm sets `app.kubernetes.io/instance` to the *name* of the installed *release*.
+
+Create and label the namespace and install the chart:
+
+```shell
+kubectl --context wds1 create ns postgres-system
+kubectl --context wds1 label ns postgres-system app.kubernetes.io/managed-by=Helm app.kubernetes.io/instance=postgres
+helm --kube-context wds1 install -n postgres-system postgres oci://registry-1.docker.io/bitnamicharts/postgresql
+```
+
+Verify that statefulset has been created in both clusters
+
+```shell
+kubectl --context cluster1 get statefulsets -n postgres-system
+kubectl --context cluster2 get statefulsets -n postgres-system
+```
+
+### Propagate helm metadata Secret to managed clusters
```

Comment:
I misunderstood earlier. It seems to me now that this subsection _is_ optional. We are talking about only something that makes `helm list` able to work on the WEC(s), _not_ about something that makes the workload functional.

Reply:
Restored to language about "optional" in latest edition.